routers/web/web.go

```diff
@@ -672,16 +674,43 @@ func RegisterRoutes(m *web.Route) {
 			})
 			http.ServeFile(ctx.Resp, ctx.Req, path.Join(setting.StaticRootPath, "public/img/favicon.png"))
 		})
-		m.Group("/{username}", func() {
-			m.Get(".png", user.AvatarByUserName)
-			m.Get(".keys", user.ShowSSHKeys)
-			m.Get(".gpg", user.ShowGPGKeys)
-			m.Get(".rss", feedEnabled, feed.ShowUserFeedRSS)
-			m.Get(".atom", feedEnabled, feed.ShowUserFeedAtom)
-			m.Get("", user.Profile)
-		}, func(ctx *context.Context) {
-			ctx.Data["EnableFeed"] = setting.EnableFeed
-		}, context_service.UserAssignmentWeb())
+		m.Get("/{username}", func(ctx *context.Context) {
```

Comment:
OK, I think this function should get moved out of web.go and into a named function within routers/web/user - and we should probably also update the `routing.UpdateFuncInfo(req.Context(), ...)` (Although we should do that without causing the funcInfo to be generated at run time.)

Reply:
In the best case the upstream issue get solved ...

the question is if we wana have that workaround now in or engineer around it properly ... or wait for upstream

Reply:
I have personally given up on waiting for `chi` to fix any issues 😉

Reply:
well that would mean **we** must fix and submit a patch upstream :laughing: